src/App.vue

```diff
@@ -4,21 +4,47 @@
       <router-view />
     </frappe-desk>
     <router-view v-else name="setup" />
+    <frappe-modal ref="modal" :show="showModal" v-bind="modalOptions" @close-modal="showModal = false"/>
```

Comment:
rebase with upstream/move-to-vue branch, this file shouldn't be in changed files